### ISSUE TITLE
Math options > Result format : crash on the simulator/not displayed properly on the device

### ISSUE THREAD

**Issue by RedGl0w** [Omega-Numworks/Omega#68]
#### Describe the bug
When I go to result format submodule, the simulator crashes
```
➜  Omega git:(omega-master) valgrind ./output/release/simulator/linux/epsilon.bin
==7735== Memcheck, a memory error detector
==7735== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==7735== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==7735== Command: ./output/release/simulator/linux/epsilon.bin
==7735== 
epsilon.bin: apps/settings/sub_menu/preferences_controller.cpp :134 : Poincare::Layout Settings::PreferencesController::layoutForPreferences(I18n::Message):  l'assertion « false » a échoué.
==7735== 
==7735== Process terminating with default action of signal 6 (SIGABRT)
==7735==    at 0x6176E97: raise (raise.c:51)
==7735==    by 0x6178800: abort (abort.c:79)
==7735==    by 0x6168399: __assert_fail_base (assert.c:92)
==7735==    by 0x6168411: __assert_fail (assert.c:101)
==7735==    by 0x36A40B: Settings::PreferencesController::layoutForPreferences(I18n::Message) (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x36A634: Settings::PreferencesController::willDisplayCellForIndex(HighlightCell*, int) (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x1A18FD: TableView::ContentView::layoutSubviews() (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x1A1931: TableView::layoutSubviews() (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x1A519A: View::setFrame(KDRect) (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x19FE11: StackViewController::ControllerView::setContentView(View*) (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x1A0152: StackViewController::setupActiveViewController() (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735==    by 0x1A0204: StackViewController::push(ViewController*, KDColor, KDColor, KDColor) (in /home/joachim/Omega/output/release/simulator/linux/epsilon.bin)
==7735== 
==7735== HEAP SUMMARY:
==7735==     in use at exit: 11,176,432 bytes in 1,123 blocks
==7735==   total heap usage: 14,111 allocs, 12,988 frees, 46,986,173 bytes allocated
==7735== 
==7735== LEAK SUMMARY:
==7735==    definitely lost: 0 bytes in 0 blocks
==7735==    indirectly lost: 0 bytes in 0 blocks
==7735==      possibly lost: 10,454,480 bytes in 2 blocks
==7735==    still reachable: 721,952 bytes in 1,121 blocks
==7735==         suppressed: 0 bytes in 0 blocks
==7735== Rerun with --leak-check=full to see details of leaked memory
==7735== 
==7735== For counts of detected and suppressed errors, rerun with: -v
==7735== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 3 from 1)
[1]    7735 abort      valgrind ./output/release/simulator/linux/epsilon.bin
```

#### Environment
 - Omega Version: 1.16.0-0

**Comment by quentinguidee** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessing result format submodule make the simulator crash](https://trello.com/c/6xjhL3m3/30-accessing-result-format-submodule-make-the-simulator-crash)

**Comment by quentinguidee** [comment]
Fixed in 1.16.1! https://github.com/Omega-Numworks/Omega/commit/44c976e030e03ee31050c01ecd4494a8ea38e5b3
